pkg/decision/composite_service.go

```diff
@@ -113,6 +114,9 @@ func (s CompositeService) GetFeatureDecision(featureDecisionContext FeatureDecis
 				convertedValue, e = strconv.ParseFloat(variableValue, 64)
 			case entities.Boolean:
 				convertedValue, e = strconv.ParseBool(variableValue)
+			case entities.JSON:
+				convertedValue = map[string]string{}
+				e = json.Unmarshal([]byte(variableValue), &convertedValue)
```

Comment:
I'm wondering now, if we should lazy unmarshal and cache the map in the variable itself the first time it is ever accessed and from there on we can just refer to that value. Seems like this could potentially be a costly operation to repeat

Reply:
Good point